### ISSUE TITLE
replay: Adding exename to -f field

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#300]
If target binary uses a lot of libraries, it'd be better to know what function is from which library.
It's possible to see it using address field but we can show the executable file name instead.
I think it'd be useful if there's `-f +exename` field for it just like `-f +task`.

**Comment by namhyung** [comment]
Do you want to see which library (or executable) the current function is from?

**Comment by namhyung** [comment]
Seems related to #278.  But adding it to output fields worries me due to its variable length.

**Comment by honggyukim** [comment]
> Do you want to see which library (or executable) the current function is from?

Exactly.

> Seems related to #278. But adding it to output fields worries me due to its variable length.

in #278, I meant to put library name in PLT hooked functions, but this issue is about general functions.  I'm also worrying about the length of exename but we may be able to limit the length somehow. I will tell you about it later.

**Comment by namhyung** [comment]
Why not doing the same for non-PLT functions too?

**Comment by honggyukim** [comment]
I thought that we can put the library name with `@libname` postfix but it could be misleading that PLT passing libraries are inside the caller binary.

**Comment by namhyung** [comment]
Hmm.. ok.  So `@libname` notation is only for PLT functions and will show library name defines the PLT function.  And this output field (how about `-f module` by the way?) will show exe/lib name just based on its address, right?

**Comment by honggyukim** [comment]
Thanks. I think `-f module` would be great!

**Comment by honggyukim** [comment]
It seems that the field `module` can also be added to `graph` command as well.

**Comment by namhyung** [comment]
Could you please check [review/libname-support-v3](https://github.com/namhyung/uftrace/compare/review/libname-support-v3) branch?

**Comment by honggyukim** [comment]
Thanks! It doesn't crash now. But could you confirm if `@libname` is for the name of its caller library or the library inside the function implemented?

**Comment by namhyung** [comment]
Good.  The `@libname` is for library implements the function, and `-f module` is for the caller.

```
     MODULE NAME   FUNCTION
          foobar | pthread_create@libpthread-2.26.so
```

**Comment by honggyukim** [comment]
That's what I expected. Thanks a lot!

**Comment by namhyung** [comment]
Merged 37c539c
